xml/System.ServiceModel/ClientBase`1+ChannelBase`1.xml

```diff
@@ -2568,7 +2707,7 @@
         <ReturnType>System.EventHandler&lt;System.ServiceModel.UnknownMessageReceivedEventArgs&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>This is a reserved event.</summary>
```

Comment:
@mconnew @carlossanlop Did this get resolved? It would be good to clean it up before we generate the IntelliSense, if possible.

Reply:
It hasn't been resolved. None of my questions have been answered. I think this might be pointing to a bigger problem in that the source that was used to update these items is incorrect as can be seen by actually looking at the documentation on docs.microsoft.com. So if this one is different than on docs and is incorrect, how many others are? And not just in the WCF documentation, across the board.

Reply:
There are two different events here that are being confused:

- [IClientChannel.UnknownMessageReceived Event](https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.iclientchannel.unknownmessagereceived?view=dotnet-plat-ext-3.1) (which is the one this summary is for)
- [ClientBase<TChannel>.ChannelBase<T>.IClientChannel.UnknownMessageReceived Event](https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.clientbase-1.channelbase-1.system-servicemodel-iclientchannel-unknownmessagereceived?view=netframework-4.8)

So I think all is well.

Reply:
How is the file supposed to be parsed? I found the explicitly implemented interface method docs further up with the expected documentation. I see that has a MemberName of `System.ServiceModel.IClientChannel.UnknownMessageReceived` due to it being explicitly implemented. This section of the documentation has the MemberName of `UnknownMessageReceived`. It doesn't show up in the documentation website as a member of ClientBase\<TChannel\>.ChannelBase\<T\>, only the explicitly implemented one. On the documentation website, why isn't there two UnknownMessageReceived entries under events, the explicitly implemented interface method (which is there) and this one? I can't see anything which suppresses this being generated as a child?  
And why is the documentation here when it's already in IClientChannel.xml? I would expect the documentation for the first link you provided to come from IClientChannel.xml, so why is it duplicated here?

Reply:
I'm not sure of the exact workings of explicitly implemented interfaces in the docs. @joelmartinez Can you help explain this?

Reply:
members are documented where they are implemented ... so even if an interface documents its member(s), every type that implements that interface/member will have its own documentation (because strictly speaking, its implementation could be unique from any other implementation when we're talking about interfaces ... or overridden members from abstract classes, etc)